src/upload/upload.controller.ts

```
@@ -1,11 +1,14 @@
-import { Controller, Post, UploadedFiles, UseInterceptors, Get, Body, Param, Query } from '@nestjs/common';
+import { Controller, Post, UploadedFiles, UseInterceptors, Get, Body, Param, Query, Sse, Res } from '@nestjs/common';
 import { FileInterceptor, FilesInterceptor } from '@nestjs/platform-express';
 import { memoryStorage} from 'multer';
 import { UploadService } from './upload.service';
+import { Subject } from 'rxjs';
+import { Response } from 'express';
 
 @Controller('upload')
 
 export class UploadController {
+    private readonly progressSuject = new Subject<any>()
```

ESLint

[error] Replace `····private·readonly·progressSuject·=·new·Subject<any>()` with `··private·readonly·progressSuject·=·new·Subject<any>();`

(prettier/prettier)